src/aws/wa_ent.yml

```diff
@@ -844,7 +858,7 @@ Resources:
                 !If [IsAWSLogDriver, "coreapp", !Ref "AWS::NoValue"]
         - Name: wa-web
           Image:
-            !Join ["", [!Ref WAEntContRegistry, "/web:", !Ref WAEntContTag]]
+            !Join ["", [!Ref WAEntContRegistry, "/web:", !Ref WAEntContTag, !Ref WAEntWebImageDigest]]
```

Comment:
Docker image ref is `<registry>/<name>:<tag>@<digest>` 

so this should be !Join ["", [!Ref WAEntContRegistry, "/web:", !Ref WAEntContTag, "@", !Ref WAEntWebImageDigest]]

Ans so on for other changes.